app/components-react/root/NewsBanner.tsx

```diff
@@ -0,0 +1,83 @@
+import { shell } from 'electron';
+import { $t } from '../../services/i18n';
+import React, { useState } from 'react';
+import { Services } from '../service-provider';
+import cx from 'classnames';
+import styles from './NewsBanner.m.less';
+import { TAppPage } from '../../services/navigation';
+import { useVuex } from '../hooks';
+
+export default () => {
```

Comment:
Use named functions `export default function MyComponent` for components. With the anonymous function, you will not see the component name in the call stack trace.